guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
@tahb also, how do you normally get changes from remote branches? - e.g. to pull down a branch for code review.